docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  leantime_web:
+    image: leantime/leantime:latest
+    restart: always
```

Comment:
For local dev/testing `restart: always` is quite annoying since it would then start these containers again on machine boot... but then again for production this might be preferable so not sure but putting it out there to discus.

Reply:
I think the primary purpose of the compose file in the main repo should be a dev environment. As such I agree that this should probably be removed.